_src/static/sass/base/_variables.scss

```diff
@@ -0,0 +1,161 @@
+/* ============================
+-------------------------------
+ 1. Grid
+ 2. Fonts
+ 3. Colors (Palette)
+ 4. Colors (Application)
+ 5. Timing
+-------------------------------
+============================ */
+
+$baseline:                      20px;
+
+
+/* ============================
+1. Grid
+---------------------------- */
+$gw-column:                     ($baseline*3);
+$gw-gutter:                     $baseline;
```

Comment:
Please remove. This is not needed now - it pertains to a grid we don't have and uses Bourbon's flex-grid (http://bourbon.io/docs/#flex-grid) which we won't be using when we get there.